schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
 	date_of_birth DATE,
 	escape_attempt INT,
 	neutered BOOLEAN,
-	weight DECIMAL
+	weight DECIMAL,
+	species VARCHAR(50)
 );
```

Comment:
- [ ] Great job implementing update animals table. However, this should the continueation of your previous milestone. You're not expected to add the species column by add but with this command. 
```
     ALTER TABLE animals ADD species varchar(255);

```
If you've created this branch from your previous branch, you should have all the previous code in this new branch which will make all the previous code inside your `schema.sql` available for this milestone. Kindly ensure you have all te previous commands available inside your schema.sql and use the above command to add new column to the animals table.